app/applications-tab/catalog/route.js

```diff
@@ -0,0 +1,27 @@
+import Ember from 'ember';
+
+export default Ember.Route.extend({
+  model: function(params) {
```

Comment:
Can you do the ajax call once and cache it so loading doesn't flash as you click through categories.